scripts/deploy-backend.sh

```diff
@@ -32,7 +32,7 @@ update_backend()
 {
     cd tdrs-backend || exit
     if [ "$1" = "rolling" ] ; then
-          bash ./scripts/set-backend-env-vars.sh "$CGHOSTNAME_BACKEND" "$CF_SPACE"
+          bash ../scripts/set-backend-env-vars.sh "$CGHOSTNAME_BACKEND" "$CF_SPACE"
```

Comment:
Found that because of the `cd tdrs-backend` a couple lines above, moving this caused an issue when running locally. Credit to @jorgegonzalez for working with me to unearth this.